addons/common/fnc_removePerFrameHandler.sqf

```diff
@@ -28,14 +28,17 @@ if (_handle < 0 || {_handle >= count GVAR(PFHhandles)}) exitWith {};
 
 [{
     params ["_handle"];
+    private _index = GVAR(PFHhandles) select _handle;
 
-    GVAR(perFrameHandlerArray) deleteAt (GVAR(PFHhandles) select _handle);
-    GVAR(PFHhandles) set [_handle, nil];
+    if (isNil "_index") exitWith {};
+    GVAR(deletedPFHIndices) pushback _index;
+
+    private _oldData = GVAR(perFrameHandlerArray) select _index;
+    _oldData set [5, false];
```

Comment:
Should 5 be 6 here to set the last array element of the `perFrameHandlerArray`entry? (The one set to "true" in `fnc_addPerFrameHandler`, `_isActiv` in `init_perFrameHandler.sqf)?

Reply:
Yes. 5 is _handle